maven-plugin/src/main/java/org/openmrs/maven/plugins/bintray/Bintray.java

```diff
@@ -0,0 +1,118 @@
+package org.openmrs.maven.plugins.bintray;
+
+import com.fasterxml.jackson.core.JsonFactory;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.type.CollectionType;
+import org.apache.commons.httpclient.HttpClient;
+import org.apache.commons.httpclient.methods.GetMethod;
+import org.apache.commons.io.FileUtils;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.util.List;
+
+public class Bintray {
+    public static List<BintrayId> getAvailablePackages(String owner, String repo){
```

Comment:
Any reason those are static? Statics are harder to test/mock and can't be overriden when extending.